OWNERS_ALIASES

```diff
@@ -188,8 +188,8 @@ aliases:
     - johnbelamaric
     - deads2k
     - wojtek-t
-# prod-readiness-approvers-emeritus:
-#    - ehashman
+  prod-readiness-approvers-emeritus:
+    - ehashman
```

Comment:
@dims @johnbelamaric - This PR is failing validation, because our testing is fetching OWNERS_ALIASES from head.
So merging this PR (if we can somehow do that manually) would fix the problem.

If we can't do that - we need to temporarily re-add Elana to prod-readiness-approvers back, and only then merge this PR.

Reply:
+1